Full Traceback (most recent call last):
  File "/opt/conda_envs/snakemake/lib/python3.10/site-packages/snakemake/__init__.py", line 671, in snakemake
    workflow.include(
  File "/opt/conda_envs/snakemake/lib/python3.10/site-packages/snakemake/workflow.py", line 1389, in include
    exec(compile(code, snakefile.get_path_or_uri(), "exec"), self.globals)
  File "/data/workflow/Snakefile", line 44, in <module>
  File "/opt/conda_envs/snakemake/lib/python3.10/site-packages/snakemake/workflow.py", line 1389, in include
    exec(compile(code, snakefile.get_path_or_uri(), "exec"), self.globals)
  File "/data/workflow/rules/plot_chanmap.smk", line 5
    Load figure names from the YAML file
IndentationError: unexpected indent

IndentationError in file /data/workflow/rules/plot_chanmap.smk, line 5:
unexpected indent
  File "/data/workflow/Snakefile", line 22, in <module>
